src/material-experimental/mdc-snack-bar/snack-bar-container.ts

```diff
@@ -188,4 +203,34 @@ export class MatSnackBarContainer extends BasePortalOutlet
       throw Error('Attempting to attach snack bar content after content is already attached');
     }
   }
+
+  /**
+   * Starts a timeout to move the snack bar content to the live region so screen readers will
+   * announce it.
+   */
+  private _screenReaderAnnounce() {
+    if (!this._announceTimeoutId) {
+      this._announceTimeoutId = setTimeout(() => {
```

Comment:
We should inject `NgZone` into this class and run the timeout outside the zone (`runOutsideAngular`) since the body here is just doing DOM manipulation and doesn't rely on Angular's change detection. This will likely impact the tests and how they work with `flush` (which is probably a good thing, since we ideally don't want to have to update tests in Google apps to land this)

Reply:
Thanks, this is very helpful. Put the `setTimeout` call in a `runOutsideAngular` block.